packages/foam-vscode/package.json

```diff
@@ -356,7 +356,9 @@
     "build": "tsc -p ./",
     "pretest": "yarn build",
     "test": "node ./out/test/run-tests.js",
+    "pretest:unit": "yarn build",
     "test:unit": "node ./out/test/run-tests.js --unit",
+    "pretest:e2e": "yarn build",
     "test:e2e": "node ./out/test/run-tests.js --e2e",
```

Comment:
This is convenient when using `test:unit/e2e`.

Reply:
On a side note, calling `yarn test` at the top level is noticeably slower than calling it here, I don't if that's because of lerna or what.

Reply:
Interesting, I guess one more reason to flatten things out